Rewrite the localization teams docs to improve legibility, and explicitly permit members who know each other (#12720)
ghostty-org/ghostty: po/README_TRANSLATORS.md

## Localization teams

Every locale has a _localization team_ consisting of the locale's _maintainers_.
These maintainers are responsible for upkeep of their locale's translations.
Anyone can contribute strings for any language's translations, and may also
volunteer to join that locale's team; a locale maintainer is only distinguished
from other speakers of the language in that:

1. Locale maintainers _declare commitment_ to updating their translations.
2. Locale maintainers are _occasionally pinged_, via GitHub's @mentions, when
   there are untranslated strings.
3. Locale maintainers are _informed of updates_ to the translations they
   maintain via GitHub's review requests.

> That's all there is to it, really. If you wish, you may leave your localization

Author: “That's all there is to it, really.” is a pretty strong way of saying the above list is exhaustive; I don't think there's any other things about locale maintainers, but I might be wrong.

Author: I feel like mentioning that a maintainer is allowed to only deal with updates, or only review, would be helpful, since currently the list is more precise than reality. I'm not sure how to word it though, since there's also the extra “there must be at least one reviewer and at least one updater” restriction.

teams at any point in the future.

> Each localization team must have at least two maintainers, who can review each

Reviewer: Is it a **must** though? We have a `vi_VN` team with only one maintainer (the other one either left or never accepted the invite), and I think we used to have more cases like this in the past. Feels like a "should" instead to me 😄

Author: > We have a `vi_VN` team with only one maintainer (the other one either left or never accepted the invite)

I had not realized; they were willing to join the team two months ago (https://github.com/ghostty-org/ghostty/pull/8912#issuecomment-4002278186), and there's been no ping for translations since then so I feel like it's more likely they just missed the invite; we should probably ask them about it (**edit**: I left a comment about it in that PR).

> I think we used to have more cases like this in the past

We did, but we did go and find more maintainers so that the PRs would actually have a chance of being merged. The only real exception is `zh_CN` (the only one which didn't have any reviewers), but there is another (sometimes, hopefully) reviewer for that too now, and the `zh_TW` maintainers helped out for previous PRs of Leah's.

In my opinion it should say “must”, unless we want to accept modifications without review.

> others' translations. Likewise, if you open a pull request for a new language,

Reviewer: Is `each others` ever correct?
```suggestion
other's translations. Likewise, if you open a pull request for a new language,
```

Author: I wasn't sure either lol.

it won't be merged until at least two people volunteer. You—the creator of the
PR—likely intend to continue helping out with them, so please make it clear

> up-front if that is not your intent.

Reviewer: Sounds like this becomes an opt-out (i.e. do we assume that someone is volunteering if they don't say otherwise in the PR desc), is that right?

Author: Yep, https://github.com/ghostty-org/ghostty/pull/12720#discussion_r3255065303.

_commitment_ to their upkeep, and being _informed_ of updates or update requests
of the translations, via GitHub's review requests or @mentions.

Each localization team must have at least two members. Otherwise, there'd be no

> one to review the work of the member when they update the translations!

Reviewer 2: This is written in a way that implies that one person has to always act in a reviewer role, but this isn't really necessarily true

```suggestion
Each localization team must have at least two members, who can review each other's
translations for each pull request.
```

Author: “Otherwise” presumes the opposite of “must have at least two members”, and given there's at least 0, that makes for either 0 or 1. I guess I was incorrect in assuming 0 implies no activity on a locale though.

Thanks for a better wording suggestion; I didn't know how to phrase it without making the whole “number of reviewers left” thing explicit.

Likewise, a pull request for a new language won't be merged until at least two
people volunteer. You—the creator of the PR—can also volunteer; if you wish to
help out in future updates, simply mention you are willing to be a part of the

> localization team in your pull request's description!

Reviewer 2: I think we should steer the first translators to take up maintainership 

```suggestion
Likewise, if you're translating Ghostty to a new language, we need at least
two volunteers (usually you, the first translator, plus another speaker
of your language) to keep these translations up to date for future updates.
```

Author: I don't think making that more obvious is too useful. The only thing I really care about my whole “consider volunteering!!!!” thing is this:

> simply mention you are willing to be a part of the localization team in your pull request's description!

In basically all cases the creator of the PR was willing to be in the localization team, but I had to ask them explicitly when they've probably already made up their mind. Excluding ones who didn't even read the document, every single new PR after that change had a “I volunteer to be in bla bla” in their PR desc :D.

I will make “if you wish to” a bit stronger though.

Author: Shrug, I had the “brilliant” idea of just negating the request (“mention if you don't want to be in the team”), hopefully this doesn't backfire from people who haven't read the document[^1].

[^1]: Since the reviewer will be asked explicitly by trag1c or I anyway, I highly doubt this would ever happen anyway.

If you are adding translations for languages that have fewer speakers, you might
worry that your pull request might never be merged, as waiting until someone

> unrelated to you reviews your PR may be probabilistically low. However, there is

Reviewer 2: Be a bit kinder towards ESL speakers and avoid mega big words

```suggestion
If you are adding translations for languages that have fewer speakers, you might
worry that your pull request will never be merged, as it is very unlikely for
someone unrelated to you to review your PR. However, there is
```

Author: > Be a bit kinder towards ESL speakers and avoid mega big words

That was half the point of rewriting the last paragraph this time lol, so oops.
